### PR TITLE
feat(storage): use finer granularity to monitor request latency

### PR DESCRIPTION
**Pull request by Little-Wallace**
Signed-off-by: Little-Wallace <bupt2013211450@gmail.com>

I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

We used to thought the iterator latency of state-store could always exceed several hundred microsecond but it is a mistake caused by metrics. In fact, in most case, it is only several micro-seconds

## Checklist

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features).
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Documentation

If your pull request contains user-facing changes, please specify the types of the changes, and create a release note. Otherwise, please feel free to remove this section.

### Types of user-facing changes

Please keep the types that apply to your changes, and remove those that do not apply.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

Please create a release note for your changes. In the release note, focus on the impact on users, and mention the environment or conditions where the impact may occur.

### CODE REVIEW

**Comment by Little-Wallace** [comment]
image

File: grafana/risingwave-dashboard.dashboard.py [hunk 1]
```diff
@@ -480,6 +480,7 @@ def metric(name, filter=None):
 
 def quantile(f, percentiles):
     quantile_map = {
+        "60": ["0.6", "60"],
```
Comment (Li0k) [inline grafana/risingwave-dashboard.dashboard.py:483]: cool, i have to questions
1. Don't we need p999 anymore?
2. Can we extend this fine-grained bucket statistics to other metrics, such as read/write duration

Comment (Little-Wallace) [reply grafana/risingwave-dashboard.dashboard.py:483]: P999 and max is duplicated. I think we only need to keep max.
In fact, if I do not close max and p999, I could even not see the value of p60 and p90....

Comment (Li0k) [reply grafana/risingwave-dashboard.dashboard.py:483]: Yes, I also often encounter this problem. This pr only modifies `create_iter_time`. Is there any update?

Comment (Little-Wallace) [reply grafana/risingwave-dashboard.dashboard.py:483]: I would change it rollback. But I still do not think we need P999 and max.

**Review by Li0k — APPROVED** [review]

**Pull request merged**